README.md

```diff
@@ -15,3 +15,17 @@ $ python -m tests
 # Run partial test with command line tools
 $ python -m tests fs_1 cond_2
 ```
+
+## Makefile
+
+You can switch scheduler by makefile flags:
```

Comment:
We should make scheduler pluggable as Linux does instead of configuring it at build time.

Reply:
It is possible to change scheduler in runtime using `sched_select` for now. But this is an easy step for running test and debugging.